### ISSUE TITLE
Plotting CSG geometries with cross_section.xml file defined in the materials.xml file

### ISSUE THREAD

**Issue by Allentro** [openmc-dev/openmc#2635]
I'm looking to plot some CSG geometries in a simple file just containing a universe of cells and nothing more.

I usually define the path to my cross-sections.xml file in my Materials() class and export to .xml, which I have checked in a simulation with all of the .xml files produced this works fine. 

I'm getting issues when plotting whereby the plotting does not appear to be finding the materials.xml file by default. 

Here's a MWE of my code here: 
```
import openmc
import matplotlib.pyplot as plt

# Cross-sections
OPENMC_CROSS_SECTIONS="<path_to_file>/cross_sections.xml"  # I have removed my actual path just for submission
materials = openmc.Materials()
materials.cross_sections = OPENMC_CROSS_SECTIONS
materials.export_to_xml()

# Surfaces
inner = openmc.Sphere(r=100)
outer = openmc.Sphere(r=200, boundary_type='vacuum')

# Regions
shell_region = +inner & -outer

# Cells
shell_cell = openmc.Cell(region=shell_region)

# Universe
universe = openmc.Universe(cells=[shell_cell])

# Plotting
colour = {shell_cell: 'blue'}
plt.show(universe.plot(basis='xz', color_by='material', colors=colour))
```

The error I'm getting is: 

`RuntimeError: No cross_sections.xml file was specified in materials.xml or in the OPENMC_CROSS_SECTIONS environment variable. OpenMC needs such a file to identify where to find data libraries. Please consult the user's guide at https://docs.openmc.org/ for information on how to set up data libraries.`

I have done a tweak in executor.py and manually defined the location of my material.xml file as the path_input to see if passing the path would solve the problem (before more appropriately implementing). That however gives an alternative error: 

`RuntimeError: No  node present in the model.xml file.`

A model.xml file is not produced in the directory.

I'm using openmc version 0.13.4.dev0. 

Am I missing a simple workaround or does anybody suggest any ways I could solve this, please?

**Comment by paulromano** [comment]
@Allentro Thanks to a recently merged PR (#2690), plotting no longer requires any cross_sections.xml to be specified, so your example above should now work. I'm going to close this issue but I'd encourage you to try it out for yourself and if you are still have issues, feel free to re-open.

**Comment by Allentro** [comment]
Great stuff - thanks Paul. I had a go with the new implementation and it works well
